samples/client/petstore/java/apache-httpclient/src/test/java/org/openapitools/client/model/SpecialCatJacksonTest.java

```diff
@@ -0,0 +1,35 @@
+package org.openapitools.client.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class SpecialCatJacksonTest {
```

Comment:
This test is in the apache-httpclient - probably fine, but the tests here are not apache client specific

Reply:
Right, I'm still learning my way around the samples... I moved it into the `java/native` sample project. Is that the best place for it?